docs/try-it-out/index.md

```diff
@@ -6,8 +6,8 @@ contentType: overview
 
 # Try it out
 
-This section gets you up and running with building workflows in n8n.
+The best way to learn n8n is by using our tutorials to get familiar with the user interface and the many different types of nodes and integrations available. Here is a selection of material to get you started:
 
-* Want a fast taster of n8n? Try the [Very quick quickstart](/try-it-out/quickstart.md).
-* Looking for a more detailed introduction? Check out the [Slightly longer introduction](/try-it-out/longer-introduction.md).
-* Prefer to work through extensive examples? Maybe the [courses](/courses/index.md) are for you.
+- Looking for a more detailed introduction? Check out the [Slightly longer introduction](/try-it-out/longer-introduction.md).
```

Comment:
We might want to change this description since it seems like a reference to the other quickstart.

Reply:
I swear i changed that already. Maybe I was dreaming